config/examples/Creality/Ender-3/BigTreeTech SKR Mini E3 3.0/Configuration_adv.h

```diff
@@ -512,7 +512,7 @@
  */
 #define USE_CONTROLLER_FAN
 #if ENABLED(USE_CONTROLLER_FAN)
-  #define CONTROLLER_FAN_PIN PB15           // Set a custom pin for the controller fan
+  #define CONTROLLER_FAN_PIN FAN2_PIN       // Set a custom pin for the controller fan
```

Comment:
There is a small risk here, if in future we decide to `#undef` pins that are not actually used. But for now this is safe, since we check for `CONTROLLER_FAN_PIN` being connected to a board fan plug in `Conditionals_post.h`.

Reply:
These fan* pin aliases make it easier to move a config between boards / upgrades, so it’d be nice if they can stick around.

Reply:
Yeah, generally it's nice to have pin defines based on the labeled pins on the board, but it quickly gets ugly if we try to do this for all pins on all boards, so this is an ok middle ground.
- name pins for their expected uses
- keep them around for other uses (avoid `#undef`ing)
- not assume a function is active just because a pin for it is defined

We used to avoid checking for enabled features in `pins.h` but now it's basic practice, and that's fine with me!